controller_interface/src/controller_interface.cpp

```diff
@@ -20,13 +20,23 @@
 namespace controller_interface
 {
 
+rclcpp::NodeOptions get_cm_node_options()
```

Comment:
as far as I can see this function is called only once. Why bothering putting this in a function then?

Also, the NodeOptions functions are reflexive, so you can make it a one-liner: `rclcpp::NodeOptions().allow_undeclared_parameters(true).automatically_declare_parameters_from_overrides(true)`.

Reply:
Oh, I didn't know that! Thanks!